lib/ExtUtils/MM_AIX.pm

```diff
@@ -53,6 +48,21 @@ sub xs_dlsyms_ext {
     '.exp';
 }
 
+sub xs_dlsyms_arg {
+    my($self, $file) = @_;
+    return qq{-bE:${file}};
+}
+
+sub init_others {
+    my $self = shift;
+    $self->SUPER::init_others;
+    # perl "hints" add -bE:$(BASEEXT).exp to LDDLFLAGS. strip that out
+    # so right value can be added by xs_make_dynamic_lib to work for XSMULTI
+    $self->{LDDLFLAGS} ||= $Config{lddlflags};
+    $self->{LDDLFLAGS} =~ s#(\s*)\S*\Q$(BASEEXT)\E\S*(\s*)#$1$2#;
```

Comment:
I'd like to see one for that. This is not overly complex, but the code is sufficiently unclear that some unpacking of what it does is merited, if only to save future readers the time to unpack.